### DISCUSSION TITLE
About northern_andes skyculture's description.en.utf8

### DISCUSSION THREAD

**Discussion by 3domennotry** [Stellarium/stellarium#3844] [Q&A]
Hello.
(Sorry, I can not speak English very well.
I'm Japanese.)

I have two questions northern_andes skyculture.

(install_dir)/skycultures/northern_andes

description.en.utf8

line29
female equinox

What is this?
Of course, I know equinox.
However, I do not know "female equinox".

line21
Through this observation they noticed that during the nightfalls of each equinox and solstice, specific stars appear in the sky, as if they were marking the climatic changes to which we are exposed, so these cosmic signals were recorded in the Andean prehispanic patrimony giving rise to the Andean geometrical forms.

If nobody feel strange by English speaker, there is no problem.
Maybe, Because simply I do not know English.

"climatic changes" is not a "seasonal change" but a "climate change theory" in Japan.

It is unrelated to "climate change theory".

I think that perhaps this is not a "climate change" but a "seasonal change".

thank you.

**Comment by gzotti** [comment]
Don't ask us about the text, ask the authors of this contribution.

**Comment by 10110111** [comment]
I think you should take the Spanish version as the reference. It was the one that appeared with the English one in the original commit (04717f4f5e4c36df326dddd0ab844300003c0aa3), apparently written by a native Spanish speaker. It's likely that the translation to English was done by the same writer, possibly suboptimally.

> However, I do not know "female equinox".

Apparently, this is a culture-specific distinction between male and female cycles (the text doesn't seem to elaborate what they are, but does mention them).

> I think that perhaps this is not a "climate change" but a "seasonal change".

Yes, from the context it does look like a seasonal change of weather. May just be a suboptimal translation to English.

**Comment by 3domennotry** [comment]
Thank you for reply.
I will compare Spanish version.

Thank you.

**Comment by 3domennotry** [comment]
I report on question No. 1.

description.es.utf8

The same words are written in this file.

line 79
equinoccio femenino

According to these website (and https://www.deepl.com/),

https://www.researchgate.net/figure/The-agro-astronomical-cycle-reflects-the-indigenous-understanding-of-the-universe-which_fig1_316335644

https://iecta.cl/wp-content/uploads/2020/03/KURI-MUYU-2.pdf
(page 7)

In this culture, March equinox is "equinoccio masculino", September equinox is "equinoccio femenino".
A grand celebration will be held on both equinoxs.

According to https://www.deepl.com/
"equinoccio masculino" translated into English is "male equinox".
"equinoccio femenino" translated into English is "female equinox".

------------------
About question No. 2

description.es.utf8

line60
cambios climáticos

According to https://www.deepl.com/
"climáticos" translated into English is "climate".

I machine-translated by "deepl".
The result is very similar.

"as if they were marking the climatic changes to which we are exposed"

description.en.utf8
as if they marked the climatic changes to which we are exposed.

In other words, this sentence already exists in the Spanish version.

I think that this sentence means "Weather changes with the seasons".

I guess I am sensitive to the term "climate change" because of the excesses and anomalies of the media and government.

thank you.
